readme/apps/sync/index.md

```diff
@@ -6,4 +6,11 @@ Currently, synchronisation is possible with Joplin Cloud, Nextcloud, S3, WebDAV,
 
 If the **terminal client** has been installed, it is possible to also synchronise outside of the user interface by typing `joplin sync` from the terminal. This can be used to setup a cron script to synchronise at a regular interval. For example, this would do it every 30 minutes:
 
-` */30 * * * * /path/to/joplin sync`
\ No newline at end of file
+` */30 * * * * /path/to/joplin sync`
+
+## Synchronisation and Migration
+
+If migrating from another application (e.g. Evernote) you will need to export your data from the old app then import it into Joplin, which can be done before or after you set up synchronization in Joplin. But once you have (a) set up synchronization and (b) imported your data, it can take some time (minutes or hours) before Joplin synchronizes all that data.
+
+**Important:** 
+The main reason to use synchronization is so that you can use the same data on more than one machine. But you should not try to set up synchronization on any additional machines until your original migrated data has been fully synchronized. Otherwise, it will look like Joplin is broken, missing some of your notes and/or notebooks. So monitor your (upload) synchronization progress via `Help >> Synchronization Status` until it shows complete; only then should you start synchronization (downloads) on other machines.
```

Comment:
I feel that can be removed since it repeats what's above? In general we avoid having redundant info as much as possible because it makes it harder to maintain the doc and to keep it structured. It also makes it more time consuming to translate it

Reply:
I completely agree that redundancy, in general, is bad. However, a bad user experience is worse. The reason I think it is necessary is that a user might land on either of those pages without ever visiting the other, and we want the caveat to be easy to stumble across.

Reply:
Unfortunately user experience suffers too when the doc has a lot of redundant information. That the user doesn't find the information they need is an argument to improve the doc structure and search, not for duplicating the same info across pages.

Reply:
I would hardly call what I have done "a lot" of redundant information. That said, you are the boss; do what you feel is best. 😁